api-docs/swagger-v1.1.yml

```diff
@@ -598,7 +600,7 @@ paths:
                 $ref: '#/definitions/total'
   /products/usages:
     get:
-      summary: list usages (categories) for a product query
+      summary: list usages (categories) for a product query, implicit usages included
```

Comment:
Het is prettig als dit een korte beschrijving is, dit wordt getoond in de endpoint overview. Past dit nog netjes? Ik vind dit eigenlijk voor zich spreken (zeker als je de output een keer bekijkt). 'k Ben een beetje bang dat onze documentatie een opeenhoping van technische details wordt ipv leesbaar te zijn.

Dit detailniveau zou ik in een _description_ oid willen zetten - alleen ondersteunt Swagger UI dat jammergenoeg (nog) niet.